aptos-move/aptos-vm/src/lib.rs

```diff
@@ -145,6 +144,15 @@ pub trait VMValidator {
     ) -> VMValidatorResult;
 }
 
+/// This trait describes the VM's simulation interfaces.
+pub trait VMSimulator {
+    /// Executes a SignedTransaction without performing signature verification.
```

Comment:
Same question re:sequence numbers.

Reply:
This does check sequence numbers, but validation does not which is still a problem because it fails in the middle of Move function

Reply:
Simulation might want to check sequence number?  It does provide a useful property that it can provide ordering in the context of a simulation for other purposes / telling people if they have the wrong sequence number (though also not that useful for the same reason)